.bazelrc

```diff
@@ -142,6 +142,10 @@ build --config=short_logs
 
 build --config=v2
 
+# Disable AWS/HDFS support by default
+build --define=no_aws_support=true
+build --define=no_hdfs_support=true
+
```

Comment:
@yongtang I think this is already being done here:
https://github.com/tensorflow/tensorflow/blob/master/.bazelrc#L255-L259

Reply:
okay I think those are specific to `build:noaws` and `build:nohdfs`. Resolving this.

Reply:
@kvignesh1420 the other place is only invoked when `noaws` is passed to bazel build, which is not the default behavior. The change in this PR is to invoke even when `noaws` is not passed to bazel build.

Reply:
yes